### PR TITLE
[java-sample] Fixing broken table & minor improvements

### PR DESCRIPTION
**Pull request by Fleeck**
image

image

### CODE REVIEW

File: java-springboot/application/src/main/resources/static/index.html [hunk 1]
```diff
@@ -18,39 +18,45 @@
...
+            <menu data-ts="Buttons" class="ts-t-t ts-buttons" style="margin-top:10px">
+                <li>
+                    <button class="ts-primary" data-ts.busy="true">
+                        <span></span>
+                    </button>
+                </li>
                 <li>
                     <button data-ts="Button" class="ts-secondary" ng-click="aside.open()">
-                        <span translate="Buttons.Open Aside">Open Aside</span>
+                        <span translate="Buttons.Open Aside">Открыть боковую панель</span>
```
Comment (jbarmash) [inline java-springboot/application/src/main/resources/static/index.html]: Please change back to English.

File: java-springboot/application/src/main/resources/static/index.html [hunk 2]
```diff
@@ -18,39 +18,45 @@
...
-                <p> This is mock data, retrieved from App's server by accessing '/demo/grid-data' URI. It uses <a href="https://tradeshift.github.io/Client-Runtime//#components/table/">Table Tradeshift UI component</a> to display the data.</p> 
+                <p> This is a mock data, retrieved from App's server by accessing '/demo/grid-data' URI. It uses <a href="https://tradeshift.github.io/Client-Runtime//#components/table/">Table Tradeshift UI component</a> to display the data.</p>
             </div>
-            <div data-ts="UI Components" id="home-table"></div>
+            <div data-ts="Table" id="home-table"></div>
         </div>
         <div id="tab2" ng-show="showTab == 2">
             <div data-ts="Note" class="note">
-                <p> Simple page demonstrating <a href="http://ui.tradeshift.com">UI Components</a> buttons and internationalization.</p>
+                <p> Simple page demonstrating <a href="http://ui.tradeshift.com">UI Components</a> such as buttons and inputs.</p>
```
Comment (jbarmash) [inline java-springboot/application/src/main/resources/static/index.html]: should be "buttons and form inputs"

File: java-springboot/application/src/main/resources/static/index.html [hunk 3]
```diff
@@ -18,39 +18,45 @@
...
+                which gives it consistent user experience with the rest of Tradeshift Apps. It was configured through Tradeshift.Developer app.
+                It calls the <a href="http://api.tradeshift.com/tradeshift/rest/external/doc">Tradeshift API</a>. Open different tabs to see the different features.
+            </p>
+            <p> The app's UI is written using Angular.js and Tradeshift UI Components.</p>
+        </div>
       </div>
         <div id="tab1" ng-show="showTab == 1">
             <div data-ts="Note" class="note">
-                <p> This is mock data, retrieved from App's server by accessing '/demo/grid-data' URI. It uses <a href="https://tradeshift.github.io/Client-Runtime//#components/table/">Table Tradeshift UI component</a> to display the data.</p> 
+                <p> This is a mock data, retrieved from App's server by accessing '/demo/grid-data' URI. It uses <a href="https://tradeshift.github.io/Client-Runtime//#components/table/">Table Tradeshift UI component</a> to display the data.</p>
```
Comment (jbarmash) [inline java-springboot/application/src/main/resources/static/index.html]: revert - remove article "a"

**Comment by jbarmash** [comment]
💯  - please correct a few minor ui feedback before merging.

File: java-springboot/application/src/main/resources/static/index.html (hunk 1 above)
Comment (Fleeck) [inline java-springboot/application/src/main/resources/static/index.html]: Oops. No idea, how it appeared here. Changed

File: java-springboot/application/src/main/resources/static/index.html (hunk 2 above)
Comment (Fleeck) [inline java-springboot/application/src/main/resources/static/index.html]: Changed

File: java-springboot/application/src/main/resources/static/index.html (hunk 3 above)
Comment (Fleeck) [inline java-springboot/application/src/main/resources/static/index.html]: Done

**Comment by Fleeck** [comment]
Changed text according to feedback, merging now.

**Pull request merged**